diff --git a/setup.py b/setup.py
index b022d2c..ef302e4 100644
--- a/setup.py
+++ b/setup.py
@@ -251,6 +251,22 @@ class PCTBuildExt (build_ext):
 
     aesni_mod_names = "Crypto.Cipher._raw_aesni",
 
+    def finalize_options(self):
+        """
+        Removes system include and library paths to avoid conflicts with cross-compilation.
+        """
+        build_ext.finalize_options(self)
+        py_include = distutils.sysconfig.get_python_inc()
+        try:
+            self.include_dirs.remove(py_include)
+        except ValueError:
+            pass
+        libdir = distutils.sysconfig.get_config_var('LIBDIR')
+        try:
+            self.library_dirs.remove(libdir)
+        except ValueError:
+            pass
+
     def run(self):
         build_ext.run(self)
 
