diff --git a/setup.py b/setup.py
index 22b4c5f..76fa946 100644
--- a/setup.py
+++ b/setup.py
@@ -41,7 +41,6 @@ setup(
     # *IMPORTANT*: Don't manually change the version here. Use `make bump`, as described in readme
     version='0.1.0',
     description="""hexbytes: Python `bytes` subclass that decodes hex, with a readable console output""",
-    long_description_markdown_filename='README.md',
     author='Jason Carver',
     author_email='ethcalibur+pip@gmail.com',
     url='https://github.com/ethereum/hexbytes',
@@ -49,7 +48,6 @@ setup(
     install_requires=[
         'eth-utils>=1.0.1,<2.0.0',
     ],
-    setup_requires=['setuptools-markdown'],
     extras_require=extras_require,
     py_modules=['hexbytes'],
     license="MIT",
