diff --git a/setup.py b/setup.py
index 52bf1c1..02be4d0 100644
--- a/setup.py
+++ b/setup.py
@@ -40,13 +40,10 @@ setup(
     # *IMPORTANT*: Don't manually change the version here. Use `make bump`, as described in readme
     version='2.0.0',
     description="""eth-typing: Common type annotations for ethereum python packages""",
-    long_description_markdown_filename='README.md',
     author='The eth-typing contributors',
     author_email='eth-typing@ethereum.org',
     url='https://github.com/ethereum/eth-typing',
     include_package_data=True,
-    setup_requires=['setuptools-markdown'],
-    python_requires='>=3.5, <4',
     extras_require=extras_require,
     py_modules=['eth_typing'],
     license="MIT",
