diff --git a/setup.py b/setup.py
index 63a6624..c0d742f 100644
--- a/setup.py
+++ b/setup.py
@@ -39,7 +39,6 @@ setup(
     # *IMPORTANT*: Don't manually change the version here. Use `make bump`, as described in readme
     version='0.2.2',
     description="""eth-account: Sign Ethereum transactions and messages with local private keys""",
-    long_description_markdown_filename='README.md',
     author='Jason Carver',
     author_email='ethcalibur+pip@gmail.com',
     url='https://github.com/ethereum/eth-account',
@@ -52,7 +51,6 @@ setup(
         "hexbytes>=0.1.0,<1",
         "eth-rlp>=0.1.2,<1",
     ],
-    setup_requires=['setuptools-markdown'],
     python_requires='>=3.5, <4',
     extras_require=extras_require,
     py_modules=['eth_account'],
